compiler/test/stdlib/number.test.gr

```diff
@@ -256,3 +256,29 @@ assert Number.sin(20) > 0.91294 && Number.sin(20) < 0.91295
 assert Number.cos(1) > 0.5403 && Number.cos(1) < 0.5404
 assert Number.cos(-1) > 0.5403 && Number.cos(-1) < 0.5404
 assert Number.cos(15) < -0.7596 && Number.cos(15) > -0.7597
+
+// Number.tan
+assert Number.tan(1) > 1.5574 && Number.tan(1) < 1.5575
+assert Number.tan(-1) > -1.5575 && Number.tan(-1) < -1.5574
+assert Number.tan(15) < -0.8559 && Number.tan(15) > -0.8560
+// Number.factorial
+assert Number.factorial(0) == 1
+assert Number.factorial(1) == 1
+assert Number.factorial(5) == 120
+assert Number.factorial(10) == 3628800
+assert Number.factorial(20) == 2432902008176640000
+// Number.clamp
+assert Number.clamp(0, 5, 0) == 0
+assert Number.clamp(-5, 5, -10) == -5
+assert Number.clamp(-5, 5, 6) == 5
+assert Number.clamp(-5, 5, 5.5) == 5
+// Number.lerp
+assert Number.lerp(0, 5, 0) == 0
+assert Number.lerp(0, 5, 0.5) == 2.5
+assert Number.lerp(0, 5, 1) == 5
+// Number.map
+assert Number.map(0, 0, 1, 100, 0) == 0
+assert Number.map(0, 0, 1, 100, 0.5) == 50
+assert Number.map(0, 0, 1, 100, 1) == 100
+assert Number.map(0, 0, 1, 100, 150) == 100
+assert Number.map(-5, 0, 0, 100, -2.5) == 50
```

Comment:
```suggestion
```
I think you also meant to remove the tests from this PR?

Reply:
🤦